compiler/fir/resolve/src/org/jetbrains/kotlin/fir/resolve/transformers/FirStatusResolveTransformer.kt

```diff
@@ -426,6 +430,14 @@ abstract class AbstractFirStatusResolveTransformer(
             isLocal = false,
             overriddenFunctions.map { it.status as FirResolvedDeclarationStatus },
         )
+        // Once the modality is determined, we can remove the body.
```

Comment:
This code is actually redundant and could be deleted.
`inline` status is not inheritable, so it checks basically the same things as the code in the raw fir builder.